CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Git LFS Changelog
 
+## 1.4.4 (21 October, 2016)
+
+### Bugs
+
+* commands,lfs/tq: Only send unique OIDs to the Transfer Queue #1600 (@ttaylorr)
+
+### Misc
+
+* Expose the result message in case of an SSH authentication error #1599
+* (@sschuberth)
```

Comment:
I'd move this under Bugs. The Misc section is for project related stuff that doesn't affect how git-lfs works, like the appveyor change.